<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Detail</class>
 <widget class="QWidget" name="Detail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Album</string>
  </property>
  <widget class="QListWidget" name="ListSongs">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>148</y>
     <width>378</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="NameLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="AuthorLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="YearLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="GenreLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddPictureButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Přidat obrázek</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddSongButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>170</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Přidat píseň</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RemoveSongButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Odebrat píseň</string>
   </property>
  </widget>
  <widget class="QLabel" name="ImageLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>ImageLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
